File: src/main/java/matteot92/prenotauncambiolook/model/entities/Servizio.java
package matteot92.prenotauncambiolook.model.entities;

import com.fasterxml.jackson.annotation.JsonPropertyOrder;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Entity
@Table(name = "servizi")
@Getter
@Setter
@ToString
@NoArgsConstructor
@JsonPropertyOrder({ "id", "descrizione", "prezzo"})
public class Servizio {
	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@Column(name = "id_servizio")
	private Long id;
	@Column(nullable=false)
	private String descrizione;
	@Column(nullable=false)
	private Double prezzo;
	
}

File: src/main/resources/frontend/src/app/models/servizio.ts
import { Ordine } from "./ordine";

export class Servizio {

  id: number;
  descrizione: string;
  prezzo: number;

  constructor(id: number, descrizione: string, prezzo: number) {
    this.id = id;
    this.descrizione = descrizione;
    this.prezzo = prezzo;
  }

}
